### ISSUE TITLE
Wallets

### ISSUE THREAD

**Issue by grand96** [StohnIO/StohnCoin#1]
Hi! Are this wallet and the wallet on the site stohncoin.org incompatible with each other?

**Comment by StohnIO** [comment]
The wallets here on github and on stohncoin.org should be compatible with each other.

**Comment by grand96** [comment]
When I launch the wallet from stohncoin.org it works. When I launch the github wallet, it doesn't connect to the nodes. (connect() to 198.199.78.228:37104 failed after select(): Connection refused (111) in debug.log)

**Comment by StohnIO** [comment]
That’s odd. That is definitely an active node. Are you a developer?

**Comment by grand96** [comment]
No, I'm not a developer, I'm just a user. Perhaps my build is not working correctly. Ok, I will use the wallet from the site stohncoin.org, it works fine
